_overviews/scala3-scaladoc/blog.md

```diff
@@ -33,4 +33,29 @@ Scaladoc loads blog if the `_blog` directory exists.
 All the blogpost filenames should start with date in numeric format matching `YYYY-MM-DD`.
 Example name is `2015-10-23-dotty-compiler-bootstraps.md`.
 
+## Structure
```

Comment:
I think maybe we should name this section something different since this part isn't so much about structure, but rather about the metadata that can be associated with a page. Also not everyone may be familiar with frontmatter so it might be good to mention that. Just an example, this could be something like:

```
## Page metadata

The blog pages in scaladoc support Yaml Frontmatter(link to this) which allows you to specify different values which will be used for metadata in your page. Here are the possible fields:
```
If list exhaustive or no? Even if it's not, what you have is better than what currently exists... nothing.